### PR TITLE
ffsend: fix build on darwin

### PR DESCRIPTION
**Pull request by onsails**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/ffsend/default.nix [hunk 1]
```diff
@@ -29,7 +29,7 @@ buildRustPackage rec {
 
   nativeBuildInputs = [ cmake pkg-config installShellFiles ];
   buildInputs =
-    if stdenv.isDarwin then (with darwin.apple_sdk.frameworks; [ CoreFoundation CoreServices Security AppKit ])
+    if stdenv.isDarwin then [libiconv] ++ (with darwin.apple_sdk.frameworks; [ CoreFoundation CoreServices Security AppKit ])
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/ffsend/default.nix:32]: While we are at it: can we remove darwin from inputs?

Comment (onsails) [reply pkgs/tools/misc/ffsend/default.nix:32]: done

**Comment by onsails** [comment]
@GrahamcOfBorg eval

File: pkgs/tools/misc/ffsend/default.nix (hunk 1 above)
Comment (onsails) [reply pkgs/tools/misc/ffsend/default.nix:32]: what do you mean by remove darwin?

**Pull request merged**